examples/annotation_label_studio/materializers/fastai_learner_materializer.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
Shouldnt this be a part of a standard fastai integration? On second thought, arent all fastai models just pytorch? So we hvae a problem like NeuralProphet here that @fa9r pointed out

Reply:
yes I have a branch open where I started working on that, but doing the whole integration with fastai + example etc was too much to do simultaneously. We can move it out later on.

And yeah it's PyTorch under the hood, but not completely so we have to handle it in a custom way.